Sources/Rendering/Misc/RemoteView/example/index.js

```diff
@@ -46,9 +46,7 @@ const config = {
 clientToConnect
   .connect(config)
   .then((validClient) => {
-    const viewStream = this.clientToConnect
-      .getImageStream()
-      .createViewStream('-1');
+    const viewStream = clientToConnect.getImageStream().createViewStream('-1');
```

Comment:
was that not already fixed?

Reply:
This branch is merging into beta. I will rebase my changes onto beta so master commits don't get mixed in. Thanks for catching that!